<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtcmWidget</class>
 <widget class="QWidget" name="RtcmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>NTRIP</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Server</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ntripServerEdit">
          <property name="text">
           <string>www.igs-ip.net</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="ntripPortBox">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Stream</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLineEdit" name="ntripStreamEdit">
          <property name="text">
           <string>SPT00</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="ntripUserEdit">
          <property name="text">
           <string>vedderb</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5" colspan="2">
         <widget class="QLineEdit" name="ntripPasswordEdit">
          <property name="text">
           <string>abc123</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ntripConnectedLabel">
          <property name="text">
           <string>Not connected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ntripDisconnectButton">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ntripConnectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Received RTCM Messages</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item row="1" column="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>1020</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLCDNumber" name="rtcm1020Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>1019</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="rtcm1019Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>GPS and GLONASS Ephemeris</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item row="1" column="3">
         <widget class="QLCDNumber" name="rtcm1006Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>1005</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>1006</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="rtcm1005Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>ARP station coordinates </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item row="1" column="3">
         <widget class="QLCDNumber" name="rtcm1003Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="rtcm1001Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>1002</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="rtcm1002Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLCDNumber" name="rtcm1004Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>1001</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>1004</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>1003</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>GPS Observaions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item row="1" column="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>1011</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="rtcm1010Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLCDNumber" name="rtcm1012Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="rtcm1009Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>1012</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>1009</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLCDNumber" name="rtcm1011Number">
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>1010</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>GLONASS Observaions</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="resetAllCountersButton">
        <property name="text">
         <string>Reset All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
